docs/README.kernel.md

```diff
@@ -231,10 +231,10 @@ KERNEL_VERSION=`make -s kernelrelease`
    issue), the `/boot/runmode/bzImage` path used by the bootloader is
    not a symbolic link, but rather the kernel itself. To avoid
    overwriting the known-good kernel, rename it appropriately by running
-   this command on the target:
+   this command:
 
    ```bash
-   test ! -h /boot/runmode/bzImage && mv /boot/runmode/bzImage /boot/runmode/bzImage-`uname -r`
+   ssh admin@$TARGET "test ! -h /boot/runmode/bzImage && mv /boot/runmode/bzImage /boot/runmode/bzImage-`uname -r`"
```

Comment:
I'm not so sure about the value of giving the ssh syntax. I think we would like to maintain that clients can run these commands however they want. And it seems like it would be more annoying to have to write in their own `TARGET` hostname every time, instead of just being able to copy the whole command into whatever ssh (or serial) connection they have open.

Reply:
We tell clients to use a `TARGET` and `KERNEL_VERSION` [here](https://github.com/ni/nilrt/blob/01444364b22aa57866b1cf66264fcb3015e4b238/docs/README.kernel.md?plain=1#L225-L228). They would still be copying without editing assuming they set those, and I would assume a client who wants to use serial or a separate ssh connection would realize they don't need the ssh portion when copying.

Additionally, some of the commands use that `KERNEL_VERSION` variable that variable wouldn't exist in the target connection unless I set it separately.

Reply:
Yeah; that's fine.